tests/core/contracts/conftest.py

```diff
@@ -444,6 +496,20 @@ async def async_math_contract(
     )
 
 
+@pytest.fixture
```

Comment:
It doesn't look like we're using this fixture anywhere other than for the fixture right below it. We could avoid creating this fixture altogether and do it all in one fixture like we did in the sync version above.

```py
@pytest_asyncio.fixture
async def async_function_name_tester_contract(async_w3, address_conversion_func):
    async_function_name_tester_contract_factory = async_w3.eth.contract(
        **FUNCTION_NAME_TESTER_CONTRACT_DATA
    )
    return await async_deploy(
        async_w3, async_function_name_tester_contract_factory, address_conversion_func
    )
    ```